Controller/PositionableController.php

```diff
@@ -75,13 +76,18 @@ private function getEntity(CRUDElement $element, $id)
 
     /**
      * @param CRUDElement $element
+     * @param Request $request
      * @return RedirectResponse
      */
-    private function getRedirectResponse(CRUDElement $element)
+    private function getRedirectResponse(CRUDElement $element, Request $request)
     {
-        return new RedirectResponse(
-            $this->router->generate($element->getRoute(), $element->getRouteParameters())
-        );
+        if ($request->query->get('redirect_uri')) {
+            $uri = $request->query->get('redirect_uri');
```

Comment:
what do you think about changing this to:
```php
$uri = $request->get('redirect_uri');
```
Redirect will also wirk then this parameter is in request attributes and post data